network-config-analyzer/NetworkConfigQuery.py

```diff
@@ -405,34 +448,31 @@ def redundant_rule_text(self, policy, rule_index, is_ingress):
             redundant_text += '\n\tNote that the action of the containing rule and the rule are different.'
         return redundant_text + '\n'
 
-    def redundant_policy_text(self, policy):
+    def redundant_policy_text(self, policy, layer_name):
         """
         Attempts to provide an explanation as to why a policy is redundant
         :param NetworkPolicy policy: A redundant policy
+        :param NetworkLayerName layer_name: The name of the layer the policy is in
         :return: A text explaining why the policy is redundant
         :rtype: str
         """
         redundant_text = f'{self.policy_title(policy)} is redundant'
         single_policy_config = self.config.clone_with_just_one_policy(policy.full_name())
         if VacuityQuery(single_policy_config).exec().bool_result:
-            if self.config.type == NetworkConfig.ConfigType.Calico:
-                redundant_text += '. Note that it allows only the default connections, as defined by profiles'
-            else:
-                redundant_text += '. Note that it allows all connections'
+            # does not allow all connections in case of host endpoints
+            redundant_text += '. Note that it allows all default connections'
```

Comment:
So maybe a more general text: "it does not change default connectivity"?

Reply:
done